code/game/machinery/syndicatebeacon.dm

```diff
@@ -61,6 +61,8 @@
 				src.updateUsrDialog()
 				spawn(rand(50,200)) selfdestruct()
 				return
+			if(2)
+				break
```

Comment:
Doesn't compile; dm does not use that syntax.

Reply:
just replace with prob(50) tbh